modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -3491,6 +3491,15 @@
 		],
 		"to": "getListType(CompanyThreadLocal.getCompanyId(), param#0#, param#1#)"
 	},
+	{
+		"classNames": [
+			"DLAppService",
+			"DLAppServiceUtil"
+		],
+		"from": "updateFileEntry(long paramName, String paramName, String paramName, String paramName, String paramName, String paramName, DLVersionNumberIncrease paramName, byte[] paramName, ServiceContext paramName)",
```

Comment:
@regiCesar21 if the pattern here expects the DLVersionNumberIncrease paramName why the LocalService doesn´t ?